torch/csrc/distributed/c10d/ProcessGroupNCCL.cpp

```diff
@@ -746,6 +746,8 @@ ProcessGroupNCCL::ProcessGroupNCCL(
       getCvarInt(TORCH_NCCL_HEARTBEAT_TIMEOUT_SEC, 60 * 10 /*10 Mins*/);
   waitTimeoutDumpInMilSec_ =
       getCvarInt(TORCH_NCCL_WAIT_TIMEOUT_DUMP_MILSEC, 2000);
+  coordDumpCheckIntervalMilSec_ =
+      getCvarInt(TORCH_NCCL_COORD_DUMP_CHECK_MS, 1000);
```

Comment:
nit - i guess we should align on MILSEC or MS consistently.  I prefer MS but i see we already have _SEC and _MILSEC so we should probably stick with that and then if we want to change them all in another PR we can

Reply:
Sure... I just don't want to make the name too long.. And MILSEC is less ambiguous.